packages/theming/default-theme/src/createAliasTokens.ts

```diff
@@ -1,4 +1,4 @@
-import { getAliasTokens, getShadowTokens } from './getTokens';
+import { getAliasTokens, getShadowTokens } from '@fluentui-react-native/theme-tokens';
```

Comment:
Please don't add all the platform tokens into the theme-tokens package. We put alias tokens for each platform in their own <platform>-theme packages to minimize the cost of importing themes, per feedback we received. Clients don't want to import theme information for platforms they don't support, but if we do this then they'll be forced to add theme tokens as a dependency for platforms they don't care about. Please put the getAliasTokens and getShadowTokens to what they had before.

Reply:
For reference: https://github.com/microsoft/fluentui-react-native/pull/1562

Reply:
Chatted offline, with what @ankraj12 was seeing this seems like a good approach. Looking through it again and making sure that the approach makes sense and won't cause the bundle size issues we had before.

Reply:
Based on conversation with @rurikoaraki , I would be updating this review resolving remaining feedbacks. This approach on forking alias tokens to be picked from theme-tokens package seems safe, and should not cause size bloat with unneeded tokens for an endpoint. More on this comment - https://github.com/microsoft/fluentui-react-native/pull/2159#discussion_r978318779

As next steps:
- I would forward merge with main, and cleanup and resolve other feedbacks.
- The changes for win32 and macOS to follow this design would be not in this PR, I would log GH issues to track this.